server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -404,6 +404,48 @@ public class ShuffleServerConf extends RssBaseConf {
       .defaultValue(-1)
       .withDescription("Shuffle netty server port");
 
+  public static final ConfigOption<Boolean> NETTY_SERVER_ENABLED = ConfigOptions
```

Comment:
We don't need this config option. If netty port is set, we should enable Netty server.